docs/adr/0003-remove-jquery-dependency.md

```diff
@@ -0,0 +1,20 @@
+# Remove jQuery and jQuery UI dependencies
+📆 **Updated**: July 7, 2022
+
+🙋🏽‍♀️ **Status** Proposed
+
+## ℹ️ Context
+- The current use of jQuery 1.7.1 is a low vulnerability (XSS)
+- There is not much use of jQuery and jQuery UI, mainly dialogs and tabs
+- It's currently responsible for the 80%+ (LOC) in `site.js`
+
+
+## 🤔 Decision
+Remove jQuery and jQuery UI dependencies. Dialogs and tabs can be implemented in CSS with a sprinkle JavaScript.
```

Comment:
Feel like we can ditch the "dialogs and tabs" bit and expand a bit.

Something like....

"We can replace existing jQuery dependencies, and build new components, by focusing on a strong foundation of markup and CSS, with JavaScript sprinkled in as necessary for enhancement."

Reply:
Also, maybe we should make it explicit for anyone later that we intend on using vanilla JS at the moment. 

Again...

"We will be using vanilla JavaScript, rather than immediately switching to another framework, to keep the performance overhead to a minimum."